packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -187,6 +191,9 @@ library LibProving {
         {
             address verifier = resolver.resolve(tier.verifierName, true);
 
+            // Auto-filled from meta value, so that no interface changes (no
+            // need to modify verifyProof)
+            evidence.usingBlob = meta.usingBlob;
```

Comment:
so the value of evidence.usingBlob is simply overwritten? Somehow I don't like the placeholder idea. Why not just pass in usingBlob directly to the verifier?

Reply:
Wanted to keep the verifyProof() as clean as possible:

```
    function verifyProof(
        uint64 blockId,
        address prover,
        bool isContesting,
        TaikoData.BlockEvidence memory evidence
    )
```

 Since others do not use the `blobHash` or `blobUsed` (like SGX). But if you prefer, i can add it back.